src/SessionTrait.php

```diff
@@ -69,7 +69,7 @@ public function memorize(string $key, $value)
         $this->startSession();
 
         if (is_callable($value)) {
-            $value = call_user_func($value, $key);
+            $value = $value($key);
```

Comment:
Isn't it possible that $value is defined in array format like `[$this, 'methodName']` ?